config/company_config.dist.yml

```diff
@@ -0,0 +1,5 @@
+sign_cipher: # add your sign cipher
+test_devices:
```

Comment:
I'd recommend to provide some example values.

In cases where we have a static set of possible values it's better to state available options in the comment.

For example:

```yaml
sign_cipher: '1234' # Your sign cipher as a string value
test_devices: # your test device numbers
    - 1234
    - 'AB12CD'
date_formats: # Possible date formats
    # see strftime() for formatting options: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes
    - '%Y-%M-%D'
    - '%D-%M-%y'
```